.github/workflows/ci.yaml

```diff
@@ -63,8 +63,10 @@ jobs:
           micromamba list
 
       - name: "Run tests"
+        env:
+          slow: ${{ fromJSON('{"false":"","true":"--runslow"}')[github.event_name == 'schedule'] }}
```

Comment:
I love this

Reply:
If this works, I am going to steal it since this pattern comes up all the time

Reply:
I'll admit it's not fully original - may have been inspired by github forums answers 😅 

That being said, if you like this, you'll love this: https://github.com/MDAnalysis/MDAKits/blob/main/.github/workflows/gh-ci-cron.yaml#L44